### ISSUE TITLE
Menetlusinfo väljad, protokollid ja struktureeritud menetlusinfo

### ISSUE THREAD

**Issue by moll** [rahvaalgatus/rahvaalgatus#174]
Menetlusinfo koosneb hetkel vaid pealkirjast ja sisust. Mitmele algatusele on lisatud menetlusinfo sisuks aga terve protokoll, ja seda osaliselt seepärast, et Riigikogu dokumentide lehe töökindluses ei olda kindel. Mitmelehelise protokolli tavateksti koopia pole hea lugemine ei Rahvaalgatuse veebilehel ega ka saadetud menetlusinfo teavituses (meilis). Ma, @moll, väidan, et saadetud menetlusinfo teavituses võiks siiski alati olla ka pealkirja kõrval sisu, et inimestele info kätte tuua ja mitte nõuda lisaliigutusi, ent olen nõus, et mitmeleheline protokolli koopia on liiast.

Esialgne lahendamist vajav probleem on seega võimaldada nii seotud dokumentide tagavarakoopiate tegemist kui ka lubada protokolli kokkuvõtte sisestamist.

Teisipäevases videokõnes sai räägitud ka ideaalsest struktureeritud kohtumise menetlusinfost, mis sisaldaks järgimisi väljasid:
- Päevakorra punktid
- Osalejad (nimi, asutus ja staatus "komisjoniliige" või "külaline")
- Ütlused ja väited (kes ütles ja mida)
- Viide Riigikogu protokollile

Kahtlustan, et niivõrd detailselt ja masinloetavalt struktureeritud infoni me niipea ei jõua. Ka ei piirdu huvitav menetlusinfo ka vaid ametlike protokollitud kohtumistega ning sisaldab ka teistes vormides sissekandeid. Pakun välja, et protokollide ja dokumentide tagavarakoopia mure saaks lahendada esialgu lihtsamalt — võimaldada menetlusinfot sisestades lisada ka faile. Nõnda jääks "sisu" väli inimloetava lühikokkuvõtte jaoks ning oleks probleemideta veebis kui ka meilis edastatav.

Praegu laivis olevate protokolli tavateksti koopiatega tuleks aga vaatamata muule midagi ette võtta (nt "Riigikogu rahanduskomisjoni istungi protokoll nr 172" @ https://rahvaalgatus.ee/initiatives/574c7dc7-efe9-42aa-8946-c4c356a36f40). Kujutan ette, et kui sündmus ümbernimetada "Riigikogu rahanduskomisjoni istung"-ks, tekstiks jätta vaid ehk istungi kuupäevad-kellajad ja lõppotsus ning viidata lõpus tervele protokollile (otselink Rahvaalgatuses hostitud PDFile), siis oleks kasutajakogemus parem kui praegu. Struktureeritud või mitte, terve pooletunnise arutelu transkripti on igal juhul pisut palju vaikimisi kuvada.

**Comment by infokujur** [comment]
AVP järgmises tegevuskavas?

**Comment by infokujur** [comment]
Luua juurde struktureeritud info (taggid)
- tulemus (nt tagasi lükatud, lahendamiseks pädevale asutusele)
- protsessi kirjeldavad märksõnad (istungite arv, avalike istungite arv)

Mõlemad tuleks kuvada nii algatuse vaates kui arhiivi kokkuvõtte tabelis

**Comment by moll** [comment]
Algatuse tulemus ka algatuste tabelisse ja filtreeritav.
Jah/ei filter ka avalik istungite jaoks. Istungite arv ka tabelisse.

**Comment by moll** [comment]
@infokujur:
> tulemus (nt tagasi lükatud, lahendamiseks pädevale asutusele)

@moll:
> Algatuse tulemus ka algatuste tabelisse ja filtreeritav.

Algatuse otsus/tulemus, mis tuleb Riigikogu liidesest, on nüüd ka filtreeritav ja kuvatud algatuste lehel menetleja all. Sai vältida eraldi tulba tegemist.

KOVi algatuste puhul on meil küll momendil üks vaba vormi lahter, mida saab adminnis muuta, aga hea oleks sellest teha loetelu lubatud väärtustest. Nõnda saaks seda masinloetavalt töödelda. On teil, @karlpallo või @infokujur, plaanis analüüsi kunagi teha, mida KOVid algatustega otsustavad, ja kuidas me seda kuvama peaks?

FYI, Riigikogu, paistab, on piirdunud järgmiste otsustega:
- TAGASTATUD
- TAGASI_LYKATUD
- LAHENDADA_MUUL_VIISIL
- ETTEPANEK_INSTITUTSIOONILE
- ETTEPANEK_VALITSUSELE
- ALGATADA_EELNOU_VOI_OTRK

**Comment by infokujur** [comment]
> KOVi algatuste puhul on meil küll momendil üks vaba vormi lahter, mida saab adminnis muuta, aga hea oleks sellest teha loetelu lubatud väärtustest. Nõnda saaks seda masinloetavalt töödelda. On teil, [@karlpallo](https://github.com/karlpallo) või [@infokujur](https://github.com/infokujur), plaanis analüüsi kunagi teha, mida KOVid algatustega otsustavad, ja kuidas me seda kuvama peaks?
> 

Ja, see nimekiri tuleb, lähtume regulatsioonist ja praktikast. Ilmselt ka 5-6 varianti.

**Comment by moll** [comment]
Uurime maksimaalselt pool tundi, kas annab kuidagi tuvastada avalikke arutelusid komisjonides. Näiteks läbi Riigikogu API kalendri järgi (kui see eksisteeris) või kuidagi seotud protokollidest vms. Avalikkuse info saaks salvestada menetlusinfo sissekande külge Rahvaalgatuse andmebaasis.

**Comment by moll** [comment]
Riigikogu istungiosakonna nõunik ütles, et kahjuks ei ole Riigikogu dokumendisüsteemis mingeid märkeid selle kohta, kas komisjoni istung oli "avalik" või mitte. Tundub, et see info liigub komisjonilt Riigikogu lehe ja meediaosakonnale käsitsi, kes siis vastavalt avalikkusele panevad videosalvestise YouTube'i. Masinloetavalt seda infot täna ei paista olevat.

Konkreetse näitena "Riigieelarve kontrolli erikomisjoni 23. sept 2024 istungid on näha:
Veebis: https://www.riigikogu.ee/riigikogu/komisjonid/riigieelarve-kontrolli-erikomisjon/paevakorrad-ja-protokollid/?startDate=23.09.2024&uuid=1c539a07-8d36-a4a4-91e7-fbfe72e9e3fb
APIs: https://api.riigikogu.ee/api/agenda/usergroup/1c539a07-8d36-a4a4-91e7-fbfe72e9e3fb?date=2024-09-23

1. Esimene istung on avalik.
   Veebis: https://www.riigikogu.ee/tegevus/dokumendiregister/toimikud/43278b3f-495d-4308-98fa-f3fcd0435915/   
   APIs: https://api.riigikogu.ee/api/volumes/43278b3f-495d-4308-98fa-f3fcd0435915
   https://www.youtube.com/watch?v=7BsS4iwmFv4
2. Teine istung on suletud.
   Veebis: https://www.riigikogu.ee/tegevus/dokumendiregister/toimikud/2f0525f9-4617-45b5-9a91-66cc92746b6a/
   APIs: https://api.riigikogu.ee/api/volumes/2f0525f9-4617-45b5-9a91-66cc92746b6a
